tests/test_scale_properties.py

```diff
@@ -1,66 +1,97 @@
-"Contains a collection of pytest tests that validates the scale properties."
+"Contains collection of pytest tests that validates the scale properties."
+
+import pytest
 
 from nidaqmx.constants import UnitsPreScaled
 from nidaqmx.errors import DaqError
 from nidaqmx.scale import Scale
 
 
-class TestScaleProperty(object):
-    """Contains a collection of pytest tests that validates the scale properties."""
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__persisted_scale__get_float64_property__returns_persisted_value(persisted_scale):
+    """Test for validating getter for float property."""
+    scale = persisted_scale.load()
+
+    assert scale.lin_slope == 2.0
+
+
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__scale__set_float64_property__returns_assigned_value(persisted_scale):
+    """Test for validating setter for float property."""
+    scale = persisted_scale.load()
+
+    scale.lin_slope = 5
+
+    assert scale.lin_slope == 5
+
+
+@pytest.mark.parametrize("persisted_scale", ["polynomial_scale"], indirect=True)
+def test__persisted_scale__get_float64_list_property__returns_persisted_value(persisted_scale):
+    """Test for validating getter for float list property."""
+    scale = persisted_scale.load()
+
+    assert scale.poly_forward_coeff == [-1.0785280415e-14, 1.0]
```

Comment:
I'm a bit surprised that coefficient 0 isn't exactly 0.0. If you compare to `[0.0, 1.0]`, does the test fail?

This property doesn't seem to return -1.0785280415e-14 on my test machine.
```
PS X:\nidaqmx-python> poetry run python
Python 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import nidaqmx
>>> s = nidaqmx.Scale('polynomial_scale')
>>> s.poly_forward_coeff
[0.0, 1.0]
>>> s.poly_forward_coeff == [0.0, 1.0]
True
```
![image](https://user-images.githubusercontent.com/17914399/229149837-d1b544a8-1fca-404d-af51-7a3e4fe5de9b.png)
![image](https://user-images.githubusercontent.com/17914399/229149882-817d78ba-0bdb-480a-837c-c18ecbb878d4.png)

If we need to do a fuzzy comparison, I think we should use `pytest.approx`: https://docs.pytest.org/en/4.6.x/reference.html#pytest-approx
```
>>> [-1.0785280415e-14, 1.0] == [0.0, 1.0]
False
>>> [-1.0785280415e-14, 1.0] == [approx(0.0), approx(1.0)]
True
```

But double precision floating point numbers can accurately reproduce integers up to about 2^53, so I wonder why this is happening.

Reply:
> I'm a bit surprised that coefficient 0 isn't exactly 0.0. If you compare to `[0.0, 1.0]`, does the test fail?
> Yes, the **test failed** when I compared with [0.0, 1.0], so I changed the comparing value to `[-1.0785280415e-14, 1.0]`

`pytest.approx` did the magic!!! Now the test is passing when comparing with `[approx(0.0), approx(1.0)]`

Reply:
I still don't understand why this fails on your machine and not my machine. I used `poetry run tox` to run the tests with Python 3.7 through 3.11 and it didn't fail.

What Python version do you have installed?

Is it 32-bit or 64-bit?

If you delete the scales from MAX and re-import the INI file, does the test pass without `approx`?

What NI-DAQmx version do you have installed?

Reply:
> I still don't understand why this fails on your machine and not my machine. I used `poetry run tox` to run the tests with Python 3.7 through 3.11 and it didn't fail.
> I use `poetry run pytest`
> What Python version do you have installed?
> Python 3.8.10 
> Is it 32-bit or 64-bit?
> 64 bit
> If you delete the scales from MAX and re-import the INI file, does the test pass without `approx`?
> I deleted the scales from MAX and re-imported again. The tests pass with `approx `
`assert scale.poly_forward_coeff == [0.0, 1.0]`  -> **PASS**
> What NI-DAQmx version do you have installed? I have NI-DAQmx 2023 Q1.
NI-DAQmx 21.8

Reply:
When I deleted and reimported the Scales in MAX, that passed the test without `approx`. Does that mean any issue with previous import in MAX or should the re-import happen for everyone using these tests?

Reply:
Okay, if re-importing the scale fixed it, then please remove `approx`. If it fails again, we'll have to revisit this.

Reply:
Sure Brad.